os/kernel/init/os_start.c

```diff
@@ -296,6 +296,10 @@ const struct tasklist_s g_tasklisttable[NUM_TASK_STATES] =
     &g_readytorun,
     TLIST_ATTR_PRIORITIZED | TLIST_ATTR_RUNNABLE
   },
+  {                                              /* TSTATE_TASK_ASSIGNED */
```

Comment:
Coding rule violations. Same as other lines

Reply:
To split different topic of change, in this commit, please modify lines you added only. And please add one more commit to modify other lines violations. (or new pr)